custom_components/ha-google-home/api.py

```diff
@@ -128,55 +130,26 @@ async def get_alarms_and_timers(self, url, device):
                         response=response,
                     )
                 )
-        return resp
 
-    async def update_google_devices_information(self):
-        """Retrieves devices from google home devices"""
-
-        _LOGGER.debug("Fetching sensor data...")
-
-        offline_devices = []
-        coordinator_data = {}
-
-        for device in await self.get_google_devices():
-            timers = []
-            alarms = []
-
-            if device.ip:
```

Comment:
It's still possible that device is offline.

Reply:
Wanted to move into a separate PR, but hey-ho, let's just keep it here :)